python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -60,41 +60,46 @@ class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
             "If not set, skips calculating convergence and runs for ``maxNumIter`` iterations",
         default=True,
     )
+    useProgressiveGain = pexConfig.Field(
+        dtype=bool,
+        doc="Allow the relative weight of the new model solution to increase with each iteration?",
+        default=True,
+    )
     doAirmassWeight = pexConfig.Field(
         dtype=bool,
         doc="Weight exposures by airmass? Useful if there are relatively few high-airmass observations.",
         default=True,
     )
     modelClampFactor = pexConfig.Field(
         dtype=float,
-        doc="Maximum relative change of the model allowed between iterations.",
+        doc="Maximum relative change of the model allowed between iterations. Set to zero to disable.",
         default=2.,
     )
-    regularizeSigma = pexConfig.Field(
+    modelClampSigma = pexConfig.Field(
         dtype=float,
-        doc="Threshold to exclude noise-like pixels from regularization.",
-        default=3.,
-    )
-    clampFrequency = pexConfig.Field(
-        dtype=float,
-        doc="Maximum relative change of the model allowed between subfilters.",
+        doc="Threshold to exclude noise-like pixels from regularization between iterations.",
```

Comment:
Is the threshold positive or negative, ie are pixels above or below the threshold excluded.

Reply:
This parameter has actually become unnecessary with the changes on this ticket, so I've simply removed it.